.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: ruby
+rvm:
+- 2.1
+
+before_script:
+ - chmod +x ./cibuild.sh # or do this locally and commit
+
+# Assume bundler is being used, therefore
+# the `install` step will run `bundle install` by default.
+script: ./cibuild.sh
+
+env:
+  global:
+  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
```

Comment:
nit: in general, I like to always have a newline at the end of every text file - no matter what kind of file it is.